tests/e2e/specs/FirstStep.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+    '@tags': ['first'],
+    'FirstStep'(browser){
+        const yesRadioSelector = '#site-container > div.columns > div.stepper-container.column > div.step-container.current.enabled > div.step-content > div > div.field.not-selected > label > span.check'
+        const noRadioSelector = '#site-container > div.columns > div.stepper-container.column > div.step-container.current.enabled > div.step-content > div > div.field.not-selected > label > span.check'
+        const attributionHeader = '#site-container > div.columns > div.stepper-container.column > div:nth-child(2) > div > h5'
+        const ccLicenseHeader = '#site-container > div.columns > div.stepper-container.column > div:nth-child(2) > div'
```

Comment:
Both attributionHeader and ccLicenseHeader basically contain the same text, because parent div of h5 (attributionHeader), and the both should be visible, whether you click yes, no, or don't click at all. What you should better check for is the text inside them: is it Attribution or Creative Commons license.